content/en/docs/demo/services/reactnativeapp.md

```diff
@@ -0,0 +1,38 @@
+---
+title: React Native App
+cSpell:ignore: typeof
+---
+
+The React Native app provides a mobile UI for users on Android and iOS devices
+to interact with the demo's services. It is built with Expo and uses Expo's
+file-based routing to layout the screens for the app.
+
+[React Native app source](https://github.com/open-telemetry/opentelemetry-demo/blob/main/src/reactnativeapp/)
+
+## Instrumentation
+
+The application uses the OpenTelemetry packages to instrument the application at
+the JS layer
+
+> **NOTE:** The JS OTel packages are supported for node and web environments.
+> While they work for React Native as well they are not currently explicitly
+> supported for that environment and may break compatibility with minor version
+> updates or require workarounds. Building more support for React Native is an
+> area of active development.
+
+The main entry point for the application is `app/_layout.tsx` where a hook is
+used to initialize the instrumentation and make sure it is loaded before
+displaying the UI:
+
+```typescript
+import { useTracer } from '@/hooks/useTracer';
+
+const { loaded: tracerLoaded } = useTracer();
+```
+
+`hooks/useTracer.ts` contains all the code for setting up instrumentation
+including initializing a TracerProvider, establishing an OTLP export,
+registering trace context propagators, and registering auto-instrumentation of
+network requests.
+
+[Instrumentation source](https://github.com/open-telemetry/opentelemetry-demo/blob/main/src/reactnativeapp/hooks/useTracer.ts)
```

Comment:
modelled from the Browser Instrumentation section in content/en/docs/demo/services/frontend.md, in that file a code snippet is included directly but given the TODOs in https://github.com/open-telemetry/opentelemetry-demo/blob/b4dc87ebeac93353106ebdbc7d29eb402e49eca8/src/reactnativeapp/hooks/useTracer.ts I thought it might be better to link out here instead, thoughts?

Reply:
Personally I don't like doc pages that link to code snippets.
But this is my opinion, I'd love to hear what the @open-telemetry/docs-approvers have to say.

Reply:
sounds good, updated here to include the snippet: https://github.com/open-telemetry/opentelemetry.io/pull/5704/commits/e41e5ea6d381a638b118e36983b95e0281a6ad2d